app/src/main/java/com/devindi/wallpaper/App.kt

```diff
@@ -85,7 +88,7 @@ class App : Application() {
             }
         }
 
-        startKoin(this, listOf(applicationModule, searchModule, mapModule), logger = koinLogger)
+        startKoin(this, listOf(applicationModule, searchModule, mapModule, dbModule), logger = koinLogger)
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/devindi/mapwall/blob/c641b62fa39c60acdf91edee9483647734123681/app/src/main/java/com/devindi/wallpaper/App.kt#L91"></span>Exceeded max line length (100) (cannot be auto-corrected)</td>
    </tr>
  </tbody>
</table>